jwql/website/apps/jwql/static/js/jwql.js

```
@@ -130,20 +138,11 @@
          'https://mast.stsci.edu/api/v0.1/Download/file?uri=mast%3AJWST%2Fproduct%2F' +
          fits_filename + '.fits');
 
-    // Show the appropriate image
```

My JS is pretty weak. But based on the testing we did, I'm assuming it's ok that these lines/variables are being removed.

---

Yes, my memory is that the structural changes and the changes to the function above make those lines redundant, and because javascript (in a web context) fails silently (i.e. if there's an exception then it just stops running at that point), having code that's not doing anything, but could still fail, is to be avoided.